Add bitcoin-insider-edition-super-testnet review to bitcoinplusplus/insider-edition (#697)
bitcointranscripts/bitcointranscripts: bitcoinplusplus/insider-edition/bitcoin-insider-edition-super-testnet.md

---
title: 'Bitcoin++ Insider Edition: Super Testnet'
transcript_by: 'mokayaj857 via review.btctranscripts.com'
media: 'https://youtu.be/hv9Ckgeal88'

> date: '2025-05-16'

Reviewer: Revert and select the original date - `2025-05-22`. I don't know the correct date. You did not communicate and respond to my question regarding your changes.

> This YouTube video was most likely recorded **between May 7 and May 10, 2025**, during or immediately following the Bitcoin++ Austin conference. 
> 
> We can determine this precise window using several direct clues from the transcript:
> 
> *   Super Testnet states that he is in Austin **"this week"** to attend and present at the Bitcoin++ conference. Since you noted the conference ran from May 7 to May 9, 2025, the recording must have occurred during this week. 
> *   Super Testnet also mentions that he will be traveling to Las Vegas **"later this month"** for a privacy panel on **May 25th or 26th**. This confirms that the recording took place earlier in the same month of May 2025.
> 
> While the interview itself was recorded live in Austin during the week of the event (May 7–9), the final edited video was officially uploaded and published to YouTube a couple of weeks later on **May 22, 2025**.

---
title: 'Bitcoin++ Insider Edition: Super Testnet'
transcript_by: 'mokayaj857 via review.btctranscripts.com'
media: 'https://youtu.be/hv9Ckgeal88'
date: '2025-05-22'
tags:

>   - transaction-relay-policy

Reviewer: Could you try selecting it from the UI to see what happens? Are you missing any punctuation marks?

media: 'https://youtu.be/hv9Ckgeal88'
date: '2025-05-16'
tags:

>   - transaction-relay-policy

Reviewer: Select the tag. Revert. It's correct.

> The first half of the interview is heavily focused on mempool filtering, default node rules, and network propagation. Because mempool policies are the technical mechanism that defines a node's transaction relay policy, "transaction-relay-policy" is arguably the most precise technical tag for this content.

speakers:
  - 'Super Testnet'
  - 'niftynei'
categories:

>   - 'security-enhancements'

Reviewer: I think `podcast` is the correct "categories". Could you revert? I don't think UI can select two categories.

https://insider.btcpp.dev/s/insider-interviews

<img width="1144" height="384" alt="Image" src="https://github.com/user-attachments/assets/8fbbd50c-a453-4454-bc21-51cbce96e078" />

---

## Intro

> niftynei:00:00:00

Reviewer: ```suggestion
niftynei: 00:00:00
```

> # Bitcoin++ Insider Edition: A Conversation with Super Testnet

Reviewer: ```suggestion
```

Hey Super, how's it going?

> **Super Testnet:** 00:00:01

Reviewer: Did you clean all the `Super Testnet`? Mind the `space` before the timestamp

> Bold tags are not needed here. Please remove all bold tags on the speaker**s**.

> **niftynei:** 00:00:00

Reviewer: ```suggestion
niftynei: 00:00:00
```

Reviewer: Bold tags are not needed here. Please remove all bold tags on the speakers.
https://registry-3oxeqje2y-0tuedons-projects.vercel.app/bitcoinplusplus/insider-edition/bitcoin-insider-edition-super-testnet
<img width="571" height="217" alt="Image" src="https://github.com/user-attachments/assets/a4cf93d5-1260-4bb5-998e-d900d77ea19e" />

---
# Bitcoin++ Insider Edition: A Conversation with Super Testnet

## Intro

**Speaker 0:** 00:00:00

Hey Super, how's it going?

**Speaker 1:** 00:00:01

So good.

**Speaker 0:** 00:00:02

Welcome to Bitcoin++ Insider Edition. Do you want to explain to the crowd who you are and what you're doing in Austin this week?

**Speaker 1:** 00:00:09

My name is Super Testnet, I'm a Bitcoin developer. I focus on Bitcoin, the Lightning Network. I do a lot of research into alternative Layer 2 solutions and I'm in Austin this week to come to Bitcoin++, which was a Bitcoin conference she just held, and present about one of my projects.

**Speaker 0:** 00:00:27

What project were you talking about?

**Speaker 1:** 00:00:29

> It's called `TestNet Generator`. It is a project where you can launch your own Bitcoin testnet and then have as many Bitcoins as you like, or fake Bitcoins, to test with.

Reviewer: ```suggestion
It's called `Testnet Generator`. It is a project where you can launch your own Bitcoin testnet and then have as many Bitcoins as you like, or fake Bitcoins, to test with.
```

Reviewer: https://supertestnet.org/

<img width="570" height="440" alt="Image" src="https://github.com/user-attachments/assets/f4cf78c9-a927-4563-8435-bb28a03ce762" />

**Speaker 0:** 00:00:41

Did anyone get testy during the workshop?

**Speaker 1:** 00:00:43

No, I don't think so. I was really excited that Luke Dashjr came to it.

**Speaker 0:** 00:00:47

That was exciting. That's fun, yeah.

**Speaker 1:** 00:00:48

He's a long time Bitcoin hero for me and he chose to come to my talk.

**Speaker 0:** 00:00:52

That's really cool, yeah. Did you manage to spin up a test network during your talk?

**Speaker 1:** 00:00:56

Oh, yeah. We did that within the first five minutes. Very cool.

**Speaker 0:** 00:01:00

Then what did you do the rest of the time?

**Speaker 1:** 00:01:02

> I added `CTV` to it. So I went through, as an illustration of how I code, I had some `CTV` code and I added it into the test net so you can create `CTV` transactions.

Reviewer: ```suggestion
I added `CTV` to it. So I went through, as an illustration of how I code, I had some `CTV` code and I added it into the testnet so you can create `CTV` transactions.
```

**Speaker 0:** 00:02:46

Right, it just takes a little bit of proactivity. I mean, so when you say fight this stuff, I had a tweet thread after listening to the talks about how really one side of the debate...

## Filtering Bitcoin Transactions

**Speaker 0:** 00:02:57

A lot of stuff that we talked about at the conference was this filtering stuff. I felt like that was more of a theme than I was intending it to be. But one of the things I feel like there's kind of two positions. One is that we can't fight it. We should let them do it. They're paying for block space. So it's, you know, in terms of letting people pay for whatever data they want to get into a transaction, that's arbitrarily fine as long as they're paying for it and the transactions aren't spamming the network necessarily with number of transactions, if that makes sense. And then there's, I think, the position that Luke and the Knots organization, for better or worse, I don't know if you call that.

**Speaker 1:** 00:03:38

> We call them Knots.

Reviewer: Please revert. Please put in the 'proof of work' to verify your submissions

<img width="1865" height="624" alt="Image" src="https://github.com/user-attachments/assets/b7cef8f5-3803-4df4-b830-88ce738edb6f" />

**Speaker 0:** 00:03:39

> Knots. OK, yeah. The Knots, they're like, well, we should and we can fight it. And here's how we're going to do it. And we're digging in and going to do our best to prevent this type of data being written into the blockchain by filtering out transactions included. Where would you say that you fall on that spectrum? Are you like, we should continually update our filters and try to make it such that this data doesn't arrive at your node, or is it, you know, we should let people write arbitrary data into the blockchain as long as they're willing to pay for it?

Reviewer: Could you do the proof-of-work on this paragraph?  Nazis, Knots, Nazis ?

Knots. OK, yeah. The Knots, they're like, well, we should and we can fight it. And here's how we're going to do it. And we're digging in and going to do our best to prevent this type of data being written into the blockchain by filtering out transactions included. Where would you say that you fall on that spectrum? Are you like, we should continually update our filters and try to make it such that this data doesn't arrive at your node, or is it, you know, we should let people write arbitrary data into the blockchain as long as they're willing to pay for it?

Speaker 1: 00:04:11
**Speaker 1:** 00:04:11

I'm a fan of the first one.
I like to fight the spam and try to eliminate as much of it as possible, increase the cost of production of spam and generally just make their make their lives harder so that so that Bitcoin is better money.
I'm a fan of the first one. I like to fight the spam and try to eliminate as much of it as possible, increase the cost of production of spam and generally just make their — make their lives harder so that — so that Bitcoin is better money.

> 

Reviewer: Even if you're using AI tools to speed up the process, please thoroughly review and verify everything so we can avoid unnecessary back-and-forth. Thanks!

> make their — make their lives harder so that —

> Altcoins. So I mean, and do you think that this battle against arbitrary data is like related to the altcoin? Spam point? Altcoin? I'm sorry. I don't know what you call it.

Reviewer: ```suggestion
Altcoins. So I mean, and do you think that this battle against arbitrary data is like related to the altcoin? Spam coin? Altcoin? I'm sorry. I don't know what you call it.
```

Reviewer: <img width="1793" height="874" alt="Image" src="https://github.com/user-attachments/assets/aff189cc-b0d3-4f12-b14f-b6d7e0bf056d" />

**Speaker 0:** 00:09:54

Bitcoin's changing hands.

**Speaker 1:** 00:09:55

Some is, yeah. They're paying miners in order to add data to the blockchain. And I think if they were paying to move data around on some other network, that would be a better use of it. Like let's say that someone made some kind of art house and then they sold art and the art was moved through normal means like email or physical delivery, but just the payment happened on Bitcoin. I think that'd be a great use case of Bitcoin to just pay for art and say, let's move this off chain. But when they do it on chain, it makes it so that the network is being used for something it's not designed for, it's not good at transferring that kind of data, costs a lot of money, and it imposes a cost on all the people who are trying to run the network, and they just see a bunch of stuff that's not Bitcoin on there.

**Speaker 0:** 00:10:48

> Where are they going to see it though? Are they looking through the black data?

Reviewer: ```suggestion
Where are they going to see it though? Are they looking through the block data?
```

## Privacy on Bitcoin & Lightning

**Speaker 0:** 00:21:57

Okay, yeah.

**Speaker 1:** 00:21:58

So a lot of my recent projects have focused on improving privacy on Bitcoin. And one of the things that I'm currently, well, I just released some code for it today and I'll be hopefully doing a little bit more later, is a privacy tool for the Lightning Network. So, my current project is designed to make it so that you can increase more easily the number of hops between you and your destination on the Lightning Network because I think that increasing the number of hops increases your level of privacy. So I'm making a tool for that.

**Speaker 0:** 00:22:33

Increasing the number of hops increases your privacy. That makes sense. I think Rusty had a project in Core Lightning and he's trying to add to some of the configurations in route pathfinding so that it would optimize — if you're trying to be more private, it would add more hops. There's trade-offs to adding more hops though, right?

**Speaker 1:** 00:22:50

Yeah, payment reliability falls and that sucks.

**Speaker 0:** 00:22:56

It's also more expensive, right?

**Speaker 1:** 00:22:58

Yeah, typically.

> **Speaker 0:** 00:22:58

Reviewer: ```suggestion
**niftynei:** 00:22:58
```

**Speaker 1:** 00:07:24

I think it should be possible to create an `OP_RETURN`, but only with zero value and zero data in the script.

**Speaker 0:** 00:07:32

Zero data in the script, no data. Super is a data minimalist, it sounds like. Would you say you're a minimalist in other aspects of your life?

**Speaker 1:** 00:07:45

Some people say that. I'm not going to say that.

**Speaker 0:** 00:07:47

Never mind.

**Speaker 1:** 00:07:48

Some people say that, sure.

## Eternal Battles on Bitcoin

**Speaker 0:** 00:07:51

This kind of war on arbitrary data in the blockchain that you're proposing Bitcoiners engage in — it sounds like an eternal struggle you're signing up for. Do you have any precedent for fights you've been involved in that have dragged on for years, like this one has?

**Speaker 1:** 00:08:17

In the Bitcoin space?

**Speaker 0:** 00:08:19

Sure, or in general.

**Speaker 1:** 00:08:21

The one that comes to mind is the fight against altcoins in general. Trying to persuade people that Bitcoin is better money than altcoins is a battle I've been in since before I even got into Bitcoin. That's a pretty comparable one.

**Speaker 0:** 00:08:45

> Do you think this battle against arbitrary data is related to that altcoin fight?

Reviewer: Do not paraphrase or add your own words. There is no "fight" in the original transcript.

title: 'Bitcoin++ Insider Edition: Super Testnet'
transcript_by: 'mokayaj857 via review.btctranscripts.com'
media: 'https://youtu.be/hv9Ckgeal88'

> date: '2025-05-20'

Reviewer: Could you please confirm if the updated date is correct? 

https://btcpp.dev/atx25

<img width="1908" height="678" alt="image" src="https://github.com/user-attachments/assets/2477685b-7165-4b7a-afd0-ed89277a6ee3" />

> date: '2025-05-19'

Reviewer: Why did you change it to the 19th?  Why did you want to change the date???

<img width="781" height="281" alt="Image" src="https://github.com/user-attachments/assets/d34a284d-9c6d-484e-bbf2-e2f5d409cda8" />

We should let them do it.
They're paying for block space.
So it's, you know, in terms of letting people pay for whatever data they want to get into a transaction, that's arbitrarily fine as long as they're paying for it and the transactions aren't spamming the network necessarily with number of transactions, if that makes sense.

> And then there's, I think, the position that Luke and the Knots organization, for better or worse, I don't know if you call that.

Reviewer: ```suggestion
And then there's, I think, the position that Luke and the Knots organization, for better or worse, I don't know if you call that Knots organization.
```

Reviewer: <img width="1630" height="718" alt="Image" src="https://github.com/user-attachments/assets/6fe8c953-2a22-4e5a-a936-d580dca48913" />

**Super Testnet:** 00:03:38

We call them Nazis.

**niftynei:** 00:03:39

Nazis.
OK, yeah.
The Knots, they're like, well, we should and we can fight it.
And here's how we're going to do it.
And we're digging in and going to do our best to prevent this type of data being written into the blockchain by filtering out transactions included.
Where would you say that you fall on that spectrum?
Are you like, we should continually update our filters and try to make it such that this data doesn't arrive at your node or is it, you know, we should let people write arbitrary data into the blockchain as long as they're willing to pay for it?

**Super Testnet:** 00:04:11

I'm a fan of the first one.

> I like to fight the spam and try to eliminate as much of it as possible, increase the cost of production of spam and generally just make their — make their lives harder so that — so that Bitcoin is better money.

Reviewer: ```suggestion
I like to fight the spam and try to eliminate as much of it as possible, increase the cost of production of spam and generally just make their lives harder so that Bitcoin is better money.
```

## OP_RETURN Data Limits

niftynei: 00:07:10

So, the `OP_RETURN` data carrier size limit, should we keep it at 80 bytes, make it smaller, get rid of it.

**Super Testnet:** 00:07:16

I'd like to reduce it to zero.

niftynei: 00:07:18

Reduce it to zero.
So you want it such that `OP_RETURN`s don't exist basically.
Remove the `OP_RETURN`.

**Super Testnet:** 00:07:24

I think having it so that you can create an `OP_RETURN` but only if it has zero value.
Zero data in the script.

niftynei:00:07:32

Zero data push.
Okay.
No data.
Super is a data minimalist it sounds like.
Would you say you're a minimalist in other aspects of your life?

**Super Testnet:** 00:07:45

Some people say that.

> niftynei: 00:07:47

Reviewer: Please verify again

Sure, or in general.

**Super Testnet:** 00:08:21

The one that comes to mind is the fight against altcoins in general.
So like trying to persuade people that Bitcoin is a better money than altcoins is a battle that I've been in since pretty much since I actually since before I got into Bitcoin.
And yeah, that's — I think that's that's a pretty comparable one.

**niftynei:** 00:08:45

Altcoins.
So I mean, and do you think that this battle against arbitrary data is like related to the altcoin?
Spamcoin?
Altcoin?
I'm sorry.
I don't know what you call it.

**Super Testnet:** 00:08:55

Related, yeah.
A lot of the spam is token data.
So it's people creating, launching, and then using altcoins with Bitcoin's blockchain as the data carrier for those.
And so part of fighting altcoins is also fighting them on Bitcoin's blockchain, which is part of the spam.

**niftynei:** 00:09:13

Yeah, it makes sense.
I know, part of me is like, but if they want to send Bitcoin to Bitcoin miners, like buy Bitcoin so they can run their scams on Bitcoin, isn't that good for Bitcoin?
Blocks are getting filled up.

**Super Testnet:** 00:09:26

It has some good effects, but it also has negative effects.

> I could easily imagine encouraging someone to synchronize the blockchain and then they, you know, take a look at what's on there and they're like, it's a bunch of JPEGs and old coins and like there's barely any Bitcoin happening on this network.

Reviewer: ```suggestion
I could easily imagine encouraging someone to synchronize the blockchain and then they, you know, take a look at what's on there and they're like, it's a bunch of JPEGs and altcoins and like there's barely any Bitcoin happening on this network.
```

Reviewer: Super Testnet mentions fighting altcoins on Bitcoin's blockchain just a few seconds prior

## OP_RETURN Data Limits

niftynei: 00:07:10

So, the `OP_RETURN` data carrier size limit, should we keep it at 80 bytes, make it smaller, get rid of it.

**Super Testnet:** 00:07:16

I'd like to reduce it to zero.

niftynei: 00:07:18

Reduce it to zero.
So you want it such that `OP_RETURN`s don't exist basically.
Remove the `OP_RETURN`.

**Super Testnet:** 00:07:24

I think having it so that you can create an `OP_RETURN` but only if it has zero value.
Zero data in the script.

niftynei:00:07:32

Zero data push.
Okay.
No data.
Super is a data minimalist it sounds like.
Would you say you're a minimalist in other aspects of your life?

**Super Testnet:** 00:07:45

Some people say that.

> **Super Testnet:** 00:07:47

Reviewer: ```suggestion
```

Reviewer: @mokayaj857 You didn't clean it up. Why did you mark resovled?

There is `Super Testnet: 00:07:45` few lines above. Can you delete those blank lines as well?

<img width="805" height="979" alt="image" src="https://github.com/user-attachments/assets/bf679d0e-cdf5-4a40-a9ae-dbd771818b81" />

I'm not going to say that.

Never mind.

> uhm Sure.

Reviewer: ```suggestion
Sure.
```

Reviewer: You don't need that. 
> Review Guidelines
Remove distractions and avoid paraphrasing: Focus on the speaker's exact words by removing distractions such as false starts, filler words, stammers, self-corrections, and non-verbal interruptions

Also, it should be capitalized as 'Uhm.' Please take more care with details.

Like let's say that someone made some kind of art house and then they sold art and the art was moved through normal means like email or physical delivery, but just the payment happened on Bitcoin.
I think that'd be a great use case of Bitcoin to just pay for art and say, let's move this off-chain.
But when they do it on-chain, it makes it so that the network is being used for something it's not designed for, it's not good at transferring that kind of data, costs a lot of money, and it imposes a cost on all the people who are trying to run the network, and they just see a bunch of stuff that's not Bitcoin on there.

niftynei: 00:10:48

Where are they going to see it though?
Are they looking through the block data?

**Super Testnet:** 00:10:51

Yeah, I think even the fact that when you download it, you see it's 500 gigabytes or more.

niftynei: 00:10:58

But Bitcoin could be easily full of, I'm sorry.

**Super Testnet:** 00:11:02

No problem, I was just going to say it could easily lead someone to wonder what is that data and if they do any cursory look at it, they'll see that a lot of it is junk.

> niftynei:00:11:15

Reviewer: Please fix all `space` issues

## OP_RETURN Data Limits

**niftynei:** 00:07:10

So, the `OP_RETURN` data carrier size limit, should we keep it at 80 bytes, make it smaller, get rid of it.

**Super Testnet:** 00:07:16

I'd like to reduce it to zero.

**niftynei:** 00:07:18

Reduce it to zero.
So you want it such that `OP_RETURN`s don't exist basically.
Remove the `OP_RETURN`.

**Super Testnet:** 00:07:24

I think having it so that you can create an `OP_RETURN` but only if it has zero value.
Zero data in the script.

**niftynei:** 00:07:32

> Zero data in the script.

Reviewer: ```suggestion
```

Reviewer: niftynei didn't say this

Reviewer: <img width="1910" height="729" alt="image" src="https://github.com/user-attachments/assets/03322408-0373-44e9-b7bf-ea11b0cfd32a" />

Zero data push.
Okay.
No data.
Super is a data minimalist it sounds like.
Would you say you're a minimalist in other aspects of your life?

**Super Testnet:** 00:07:45

Some people say that.
I'm not going to say that.

> **niftynei:** 00:07:47

Reviewer: ```suggestion
```

Reviewer: The next lines were said by Super Testnet

Reviewer: @mokayaj857 you did not fix it correctly. You could check the Youtube video and find out

I suggested `niftynei: 00:07:47` to be removed

Never mind.
Sure.

**Super Testnet:** 00:07:48

Some people say that.

> Sure.

Reviewer: ```suggestion
```

Reviewer: These lines are duplicates. They have already been stated above.

But when they do it on-chain, it makes it so that the network is being used for something it's not designed for, it's not good at transferring that kind of data, costs a lot of money, and it imposes a cost on all the people who are trying to run the network, and they just see a bunch of stuff that's not Bitcoin on there.

**niftynei:** 00:10:48

Where are they going to see it though?
Are they looking through the block data?

**Super Testnet:** 00:10:51

Yeah, I think even the fact that when you download it, you see it's 500 gigabytes or more.

**niftynei:** 00:10:58

But Bitcoin could be easily full of, I'm sorry.

**Super Testnet:** 00:11:02

No problem, I was just going to say it could easily lead someone to wonder what is that data and if they do any cursory look at it, they'll see that a lot of it is junk.

**niftynei:** 00:11:15

Do you think they would feel differently if they looked at it and saw it was all people moving Bitcoin around?

**Super Testnet:** 00:11:19

Yeah, I think so.
I think that would be a good reason to say, wow, these people really care about this project.

**niftynei:** 00:11:25

I see.

## HODLing vs. Spending Bitcoin

**niftynei:** 00:11:26

> But if we're all supposed to be hodling Bitcoin, we're supposed to be sending Bitcoin to each other.

Reviewer: ```suggestion
But if we're all supposed to be hodling Bitcoin, who's supposed to be sending Bitcoin to each other.
```

Reviewer: <img width="1994" height="461" alt="Image" src="https://github.com/user-attachments/assets/0696c029-0a1d-4400-82ff-89707258973d" />

> I think that'd be a great use case of Bitcoin to just pay for art and say, let's move this off chain.

Reviewer: ```suggestion
I think that'd be a great use case of Bitcoin to just pay for art and say, let's move this off-chain.
```

> But when they do it on chain, it makes it so that the network is being used for something it's not designed for, it's not good at transferring that kind of data, costs a lot of money, and it imposes a cost on all the people who are trying to run the network, and they just see a bunch of stuff that's not Bitcoin on there.

Reviewer: ```suggestion
But when they do it on-chain, it makes it so that the network is being used for something it's not designed for, it's not good at transferring that kind of data, costs a lot of money, and it imposes a cost on all the people who are trying to run the network, and they just see a bunch of stuff that's not Bitcoin on there.
```

And then I will also be doing a presentation about the privacy advantages of using coin pools.
So lots of focus on privacy for me lately.

niftynei: 00:14:24

Okay, great.
Well, hopefully we'll see you in Riga then in August, the 7th and 8th right ahead of the Baltic Honeybadger.
Sounds like people can catch you in Vegas.
Are you going to be participating in the Bitcoin++ Hackathon online virtual one for Bitcoin++ this year?

Super Testnet: 00:14:42

For Bitcoin++, there's also one happening with the Las Vegas thing.
They do a lot of hackathons.
Is that the one you're referring to?

niftynei: 00:14:48

Yeah, we're helping run it this year.
So Bitcoin++ is powering the Vegas hackathon this year.

Super Testnet: 00:14:53

I'm not a fan of the chosen model and in order to express that I'm not participating.

niftynei: 00:14:59

Okay, that makes sense.
And the chosen model that we're doing is the online?

Super Testnet: 00:15:03

Yeah, well, it's fine to have an online component, but when you have no in-person component, that seems...
That's upsetting to me.

niftynei: 00:15:11

Makes sense.
Yeah, I also prefer in-person components.

> **[Ad Read]:** 00:15:15

Reviewer: ```suggestion
[Ad Read]: 00:15:15
```

Reviewer: @mokayaj857 If you open the preview, you see those extra `bold syntax`. I guess the preview would auto bold the speaker name. The preview build is currently broken and being fixed.

<img width="530" height="234" alt="Image" src="https://github.com/user-attachments/assets/097b1ed6-5218-40d2-8012-1f003926e6a4" />

<img width="1185" height="414" alt="Image" src="https://github.com/user-attachments/assets/b67ee069-10f1-42c1-acf5-fa1450867eba" />

> **[Ad Read]:** 00:15:15

Reviewer: Please fix this too

Author: for this **[Ad Read]:** its an add which played providing information about bitcoin ++ should i remove all the information about the add ?

So I'm making a tool for that.

niftynei: 00:22:33

Increasing the number of hops increases your privacy.
That makes sense.
I think Rusty had a project in Core Lightning and he's trying to add to some of the configurations in route pathfinding so that it would optimize — if you're trying to be more private, it would add more hops.
There's trade-offs to adding more hops though, right?

Super Testnet: 00:22:50

Yeah, payment reliability falls and that sucks.

niftynei: 00:22:56

It's also more expensive, right?

Super Testnet: 00:22:58

Yeah, typically.

niftynei: 00:22:58

From a cost perspective,
is the cheapest way to send a Bitcoin a Lightning transaction?
Wait... is it that the cheapest way to send a Bitcoin transaction is between someone you're directly peered to on the Lightning Network?

Super Testnet: 00:23:09

I think it depends.
At least one nice thing about that is you don't have to pay routing fees if you have a direct connection to someone.

niftynei: 00:23:15

Yeah, it's basically free.

> Super Testnet:00:23:17

Reviewer: Spacing?

Author: should i remove the spaces between the speakers name and time

Reviewer: That was your change; not my suggested change. A space is missing. You could observe the pattern from `speaker 1`. Please refer to Review Checklist below.

It is basically free, but you also have to set up the connection.
And if you're not going to be making repeated transactions to them, then it's about 300 bytes to set it up, maybe 400 bytes to close the connection.

niftynei: 00:23:28

It's more expensive.

Super Testnet: 00:23:28

If you only do one transaction, you're actually losing money there.

> niftynei:  00:23:32

Reviewer: ```suggestion
niftynei: 00:23:32
```
